cfme/automate/buttons.py

```diff
@@ -539,41 +558,45 @@ class ButtonGroupCollection(BaseCollection):
     TEMPLATE = "VM Template and Image"
     VM_INSTANCE = "VM and Instance"
 
-    def instantiate(self, text, hover, type, image=None):
+    def instantiate(self, text, hover, type, image=None, display=None, icon_color=None):
         if image:
             pass
         elif self.appliance.version < '5.9':
```

Comment:
This can be removed now . We are not checking for versions less that 5.9 anymore

Reply:
@sshveta still we have 5.8 branches. I will add card to remove it.

Reply:
:+1: